frame/democracy/src/lib.rs

```diff
@@ -544,6 +556,10 @@ decl_error! {
 		WrongUpperBound,
 		/// Maximum number of votes reached.
 		MaxVotesReached,
+		/// The provided witness data is wrong.
+		InvalidWitness,
```

Comment:
self/random note that some other modules can generally use this terminology: Some data that is passed in for sanity/weight should be called witness data.